src/azure-cli/azure/cli/command_modules/acs/decorator.py

```diff
@@ -0,0 +1,4503 @@
+# --------------------------------------------------------------------------------------------
```

Comment:
What's this file for?

Reply:
removed. accidentally got there when resolve merge conflict